.circleci/config.yml

```diff
@@ -11,6 +11,12 @@ jobs:
 
     steps:
       - checkout
+      - run:
+          name: Check for large files
+          command: |
+            set +e
+            find . -size +1Mc -not -path './.git/*' | grep .
+            (( $? == 1 ))
```

Comment:
`find` will output matching files. `grep .` will match any file, so if `find` outputs any files, `grep` will return with an exit status of 0. We want the opposite though: we want there to be no matching files. In that case, `grep` returns with an exit status of 1. In other words, we want the command as a whole to succeed only if `grep` returns 1.

It [looks like](https://circleci.com/docs/configuration-reference/#default-shell-options) CircleCI runs the command with `-e`, which would cause the command to fail if `grep` returned with an exit status of 1. That's why I run `set +e`.